src/integrationTest/sam.test.ts

```diff
@@ -264,88 +254,100 @@ describe('SAM Integration Tests', async () => {
-                        undefined,
-                        'unexpected debug session in progress'
-                    )
-
-                    const codeLens = await getDebugLocalCodeLens(samAppCodeUri, scenario.language)
-                    assert.ok(codeLens, 'expected to find a CodeLens')
-
-                    const debugSessionStartedAndStoppedPromise = new Promise<void>((resolve, reject) => {
-                        testDisposables.push(
-                            vscode.debug.onDidStartDebugSession(async startedSession => {
-                                const sessionValidation = validateSamDebugSession(
-                                    startedSession,
-                                    scenario.debugSessionType
-                                )
-
-                                if (sessionValidation) {
-                                    await stopDebugger()
-                                    throw new Error(sessionValidation)
-                                }
-
-                                // Wait for this debug session to terminate
-                                testDisposables.push(
-                                    vscode.debug.onDidTerminateDebugSession(async endedSession => {
-                                        const endSessionValidation = validateSamDebugSession(
-                                            endedSession,
-                                            scenario.debugSessionType
-                                        )
-
-                                        if (endSessionValidation) {
-                                            throw new Error(endSessionValidation)
-                                        }
-
-                                        if (startedSession.id === endedSession.id) {
-                                            resolve()
-                                        } else {
-                                            reject(new Error('Unexpected debug session ended'))
-                                        }
-                                    })
-                                )
-
-                                // wait for it to actually start (which we do not get an event for). 800 is
-                                // short enough to finish before the next test is run and long enough to
-                                // actually act after it pauses
-                                await sleep(800)
-                                await continueDebugger()
-                            })
-                        )
-                    })
-
-                    await vscode.commands.executeCommand<any>(
-                        codeLens.command!.command,
-                        ...codeLens.command!.arguments!
-                    )
-
-                    await debugSessionStartedAndStoppedPromise
-                }).timeout(TIMEOUT * 2)
+                // it.skip('invokes the Run Local CodeLens', async () => {
```

Comment:
why not leave this stuff as `it.skip` ? better to avoid diff churn where possible. @gsweetwood is planning to work on restoring these SAM tests

Reply:
The linter doesn't like the removal of `getRunLocalCodeLens` and `getDebugLocalCodeLens`

Reply:
Swapped.